### ISSUE TITLE
Dpp should generate the correct type,mangling size and alignment for aggregate ref member

### ISSUE THREAD

**Issue by Laeeth** [atilaneves/dpp#150]
your code might work but dpp offset probably doesn't
https://github.com/rust-lang/rust-bindgen/issues/1443

**Comment by atilaneves** [comment]
> That's because you are ignoring my PR

Not at all, I'm working through the previous one, figuring out how best to support the usecase, making it happen then moving on to the other two.

**Comment by Laeeth** [comment]
Okay, tku :)
